manubot/cite/doi.py

```diff
@@ -7,6 +7,32 @@
 from manubot.cite.pubmed import get_pubmed_ids_for_doi
 from manubot.util import get_manubot_user_agent
 
+from manubot.cite.types import Handle, CSL_Item
+
+class DOI(Handle):
+    def canonic(self):
+        self.identifier = self.identifier.lower()
```

Comment:
switch order of lowercasing and expanding so lowercasing is done last

Reply:
cae3e5f